src/clj/rems/api/services/organizations.clj

```diff
@@ -60,9 +61,10 @@
   (organizations/update-organization! (:organization/id org)
                                       (fn [db-organization]
                                         (let [organization-owners (set (map :userid (:organization/owners db-organization)))
-                                              organization-owner? (contains? organization-owners userid)]
+                                              organization-owner? (contains? organization-owners userid)
+                                              owner? (contains? context/*roles* :owner)]
                                           (merge db-organization
-                                                 (remove-keys (if organization-owner?
+                                                 (remove-keys (if (and organization-owner? (not owner?))
```

Comment:
would this be simpler to express as `(if-not owner? ...)`? as in only owner role can update owners but roles can also include other roles (like organization owner)

Reply:
Hmm, I guess it would. Let's however change this in the issue #2828, where we need to check whether it's an owner or organization owner of the same organization.